tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -971,25 +970,23 @@ tensorflow::Status ConvertAfterShapes(ConversionParams& params) {
                                 &graph, alloc.get(), &engine_nodes);
     // If status is ok, we successfully added the node to the graph and can
     // remove segment ops. Otherwise graph is not modified.
-    string msg = StrCat("Engine ", engine.engine_name, " creation for segment ",
-                        i, ", composed of ",
+    string msg = StrCat("TensorRT node ", engine.engine_name, " added for segment ",
+                        i, " consisting of ",
                         converted_segments.at(i).first.size(), " nodes");
-    if (VLOG_IS_ON(1)) {
-      StrAppend(&msg, " (");
-      for (const string& node_name : converted_segments.at(i).first) {
-        StrAppend(&msg, node_name, ", ");
-      }
-      StrAppend(&msg, ")");
-    }
     if (status.ok()) {
       LOG(INFO) << msg << " succeeded.";
       for (auto node_name : converted_segments.at(i).first) {
         graph.RemoveNode(node_map.at(node_name));
       }
     } else {
       // Graph is not modified.
-      LOG(WARNING) << msg << " failed: " << status << ". Skipping...";
+      LOG(WARNING) << msg << " failed: " << status << ". Fallback to TF...";
+    }
+    msg = "Segment consists of nodes: ";
```

Comment:
Please add `if (VLOG_IS_ON(1)) ` so we don't do this by default.

Reply:
Done